rootfs/entrypoint.sh

```diff
@@ -62,5 +62,22 @@ COMMAND="autossh "\
 
 echo "[INFO ] # ${COMMAND}"
 
-# Run command
-exec ${COMMAND}
+# Run autossh command in background and save pid
```

Comment:
Please write a test for this in a new container, you may call it the `local-post-script` container if you'd like.

The test container should exit with `0` when you have proven the post_script ran successfully and return non-zero when the script failed to run.